src/plugins/midi-piano/midi-piano.yml

```diff
@@ -0,0 +1,16 @@
+name:
+  en: Midi piano
+  de: Midi-Klavier
+midiTrackTitle:
+  en: Track title
+  de: Stück-Titel
+pianoKeyRange:
+  en: Key range
+  de: Tastenumfang
+confirm:
+  en: Confirm
+  de: Bestätigen
```

Comment:
You could remove the translation for "confirm" from here, as there is a shared translation in `common.yml`. This means that in `src/plugins/midi-piano/midi-piano-editor.js` on line 160, you can replace `t('midiPiano:confirm')` with `t('common:confirm')`